assets/js/pages/ClusterDetails/clusterOperations.js

```diff
@@ -134,6 +140,42 @@ export const getClusterHostOperations = curry(
         setOperationModelOpen({ open: true, operation: PACEMAKER_DISABLE });
       },
     },
+    {
+      value: 'Start host',
```

Comment:
nitpick:
Maybe `Start cluster host` ?

Reply:
Well, we are acting on a host inside a cluster page, I guess it's redundant

Reply:
@abravosuse @jagabomb what do you think?
Just to double check, i wouldn't be surprised of creating false expectations :sweat_smile: 
I would at least put some additional information in the modal. Something making super clear that we are talking about start/stop the cluster in that host.
Something like `Start cluster in host` or whatever